libs/bead-brain/config.ts

```diff
@@ -0,0 +1,37 @@
+
+namespace config {
+
+    export const PIN_ACCELEROMETER_SDA = DAL.PA00;
+    export const PIN_ACCELEROMETER_SCL = DAL.PA01;
+    export const PIN_ACCELEROMETER_INT = DAL.PA13; 
+
+    export const PIN_A1 = DAL.PA07;
+    export const PIN_A2 = DAL.PA06;
+    export const PIN_A3 = DAL.PA05;
+    export const PIN_A4 = DAL.PB08;
+    export const PIN_A5 = DAL.PB09;
+    export const PIN_A6 = DAL.PA04;
+    export const PIN_A7 = DAL.PB03;
+
+    export const PIN_D1 = DAL.PA07;
+    export const PIN_D2 = DAL.PA06;
+    export const PIN_D3 = DAL.PA05;
+    export const PIN_D4 = DAL.PB08;
+    export const PIN_D5 = DAL.PB09;
+    export const PIN_D6 = DAL.PA04;
+    export const PIN_D7 = DAL.PB03;
+    export const PIN_D13 = DAL.PA17;
+
+    export const PIN_TX = DAL.PB08;  
+    export const PIN_SCK = DAL.PA21;
+    export const PIN_DISPLAY_CS= DAL.PB22;
```

Comment:
do we need this?

Reply:
eh, not really right now...i was keeping it for when a display is hooked up to this eventually